src/web/components/App.tsx

```diff
@@ -316,9 +318,13 @@ export const App = ({ CAPI, NAV }: Props) => {
                     />
                 </Lazy>
             </Portal>
-            <Portal root="cmp">
-                <CMP cmpUi={CAPI.config.cmpUi} />
-            </Portal>
+            {!CAPI.config.cmpUi && (
+                <Portal root="cmp">
+                    <Suspense fallback={<div>Loading...</div>}>
```

Comment:
Unsure if we should use `Suspense` here as I assume we don't need a fallback loading state